test/image/mocks/heatmap-log-scale.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
Please rename this mock and baseline to start with `zz-` as `test/image/mocks/zz-heatmap-log-scale.json`
That would help avoid a bug in our current parallel image test system.

Reply:
renamed